elasticsearch/kontena.yml

```diff
@@ -0,0 +1,78 @@
+stack: kontena/elasticsearch
+version: 0.1.0
+description: Elasticsearch
+expose: elastic
+variables:
+  instances:
+    type: integer
+    from:
+      prompt: How many instances?
+  reset_passwords:
+    type: boolean
+    from:
+      prompt: Reset all default passwords during initial stack installation?
+  elastic_password:
+    only_if: reset_passwords
+    type: string
+    from:
+      vault: ELASTIC_PASSWORD
```

Comment:
Best practice with secrets: `${STACK}-password`

Reply:
Yes, but there's kinda problem as both Kibana and Fluentd stacks also need this :) So in this case it really is a "cross-stack" secret

Reply:
Ok, but then it's `ELASTIC_PASSWORD` vs `elastic-password` ... other stacks are using non env-ish key names (for better readability).